taxonomies/languages.txt

```diff
@@ -11324,50 +11324,51 @@ language_code_2:en:mh
 language_code_3:en:mah
 wikidata:en:Q36280
 
-en:Modern Greek, Modern Greek language, el, el, gre
-be:Новагрэчаская мова
+en:Greek, Greek language, Modern Greek, Modern Greek language, el, el, gre
+be:Грэчаская мова, Новагрэчаская мова
 ca:Grec demòtic
 chr:ᎠᏂᎪᎢ ᎧᏬᏂᎯᏍᏗ
-cs:Novořečtina, Moderní řečtina
-da:Nygræsk
-de:Neugriechisch, Modernes Griechisch, Neugriechische Sprache
-el:Νέα ελληνική γλώσσα, Νέα ελληνική, Νεοελληνική, Νεοελληνική Γλώσσα, Νέα ελληνικά
+cs:Řečtina, Novořečtina, Moderní řečtina
+da:Græsk, Nygræsk
+de:Griechisch, Griechische Sprache, Neugriechisch, Modernes Griechisch, Neugriechische Sprache
+el:Ελληνικά, Ελληνική γλώσσα, Νέα ελληνική γλώσσα, Νέα ελληνική, Νεοελληνική, Νεοελληνική Γλώσσα, Νέα ελληνικά
 es:griego moderno, Idioma griego moderno
-fi:Nykykreikka
-fo:Nýgrikskt mál
+fi:Kreikan kieli, Nykykreikka
```

Comment:
```suggestion
fi:Kreikka, Kreikan kieli, Nykykreikka
```
Calling it just 'Kreikka' shouldn't be a problem when we know it refers to the language.